dt.go

```diff
@@ -0,0 +1,16 @@
+// user/bin/go run $0 $@ ; exit
+
+// English dictionary and thesaurus lookup CLI tool
+package main
+
+import (
+	"github.com/Johnsoct/dicthesaurus/business"
+	"github.com/Johnsoct/dicthesaurus/presentation"
+)
+
+func main() {
+	lookupValue := business.GetSubCommand()
+	response := business.GetDefintionInBytes(lookupValue)
+	data := business.UnmarshaledJSON(response)
+	presentation.Print(data)
```

Comment:
From a high-level perspective, this just doesn't tell the reader what steps are being taken in clear, simple ways:

1. GetLookupValue
2. GetDefinition
3. UnmarshalResponse
4. Print